test/OpenTelemetry.Instrumentation.Dependencies.Tests/HttpClientTests.Basic.netcore31.cs

```diff
@@ -100,8 +100,8 @@ public async Task HttpDependenciesInstrumentationInjectsHeadersAsync()
         [Fact(Skip = "TODO: Reenable once custom format support is added")]
         public async Task HttpDependenciesInstrumentationInjectsHeadersAsync_CustomFormat()
         {
-            var textFormat = new Mock<ITextFormat>();
-            textFormat.Setup(m => m.Inject<HttpRequestMessage>(It.IsAny<SpanContext>(), It.IsAny<HttpRequestMessage>(), It.IsAny<Action<HttpRequestMessage, string, string>>()))
+            var textFormat = new Mock<ITextFormatActivity>();
```

Comment:
Can we enable this test now with this change?

Reply:
had enabled it, but missed pushing that commit!